packages/sdk/src/sdk/types/SolanaAddress.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from 'zod'
+
+export const SolanaAddressSchema = z.custom<string>((val) => {
```

Comment:
we do soemthing different elsewhere for this, you can do new PublicKey(...) and try/catch it.

I think it's a better check because it sees if it lies on the curve or not

https://github.com/AudiusProject/audius-protocol/blob/e11cacb28f3e351de8a507d466fb88d508ceba5e/packages/web/src/services/solana/solana.ts#L18

Reply:
Oh nice, didn't know that existed. Will update.